kubernetes/config-maps/nginx-storage/default.conf.j2

```diff
@@ -207,14 +208,34 @@ server {
 
             os.rename("/srv/storage/" .. tmp_file_name, "/srv/storage/" .. header_path_with_filename)
             os.remove("/srv/storage/" .. tmp_file_name .. ".shactx")
+
+            -- Send confirmation to the conductor after uploaded file.
+            local http = require "resty.http"
+            local httpc = http.new()
+            local res, err = httpc:request_uri("{{ upload_report_url }}" .. header_path_with_filename, {
+            method = "POST",
+            })
+
+            if (res.status < 200 or res.status >= 300) then
+                ngx.status = 502
+                error_response = '{"error_message":"Failed to report upload. Conductor responded with status {' .. res.status .. '}", "conductor_message":"' .. res.body .. '"}'
+                ngx.say(error_response)
+                -- Return response body in nginx log for debugging purposes
+                ngx.log(ngx.ERR, error_response)
+                return ngx.exit(502)
+            end
+
             -- Nginx big upload module adds headers with checksum and temporary file name to its response.
             -- We don't want to return them to the client.
             -- FIXME: Setting it to an empty string does not remove the headers. For now we set it to a
             -- single space to work around this but we should really just find a way to remove those headers.
             ngx.header["X-SHA1"]       = " "
             ngx.header["X-Session-Id"] = " "
             ngx.header.content_type    = "application/json; charset=utf-8"
-            ngx.say('{"message": "File was successfully uploaded"}')
+            successfully_response      = '{"message": "File was successfully uploaded"}'
+            ngx.say(successfully_response)
+            ngx.log(ngx.NOTICE, successfully_response)
```

Comment:
Adding more logging also could be a separate commit.

Reply:
There are now three commits for 36 line of code. Are you sure that we want add next commit with one or two line of code?

Reply:
Yes. It's not about the size. Even a single-line change is a good commit if it's something unrelated to the other lines.

You make a separate commit because you want to show that it's a single, coherent change, not because your change is too big.

You should generally make smaller commits.